entry.go

```diff
@@ -219,7 +219,6 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 			stream, err = session.AcceptStream()
 			if err != nil {
 				log.Println("Close connection:", err)
-				te.close()
```

Comment:
Why not close te here? When accept returns error, it means the connection is already broken and cannot be used anymore.

Reply:
Entry will closed at L308, this line will cause panic if exit close.The code in master branch doesn't call `te.close` either.

Reply:
A more robust solution is to allow close() to be called multiple times. Example: https://github.com/nknorg/nkn-sdk-go/blob/master/client.go#L168

Reply:
We need to add back te.close() here